update_npm_dependencies.rb

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'json'
+
+def package_name(npm_module_name)
+  "nodejs-#{npm_module_name}"
+end
+
+def package_dir(npm_module_name)
+  "packages/foreman/#{package_name(npm_module_name)}"
+end
+
+def package_exist?(npm_module_name)
+  File.directory?(package_dir(npm_module_name))
+end
+
+def current_version(npm_module_name)
+  if package_exist?(npm_module_name)
+    spec_file = "#{package_dir(npm_module_name)}/#{package_name(npm_module_name)}.spec"
+    version_line = File.readlines(spec_file).grep(/^\W*Version:.*/).first
+    version_line.gsub(/^\W*Version:\W*/, '').strip
+  end
+end
+
+def bundle_strategy(npm_module_name)
+  if package_exist?(npm_module_name)
+    # new packages default to bundle
+    'bundle'
+  else
+    dir = package_dir(npm_module_name)
+    zipped_dependencies = Dir["#{dir}/*.tgz"]
+    if zipped_dependencies.length > 1
+      'bundle'
+    else
+      'single'
+    end
+  end
+end
+
+def detect_action(npm_module_name, details)
+  if package_exist?(npm_module_name)
+    if details['version'] == current_version(npm_module_name)
+      :skip
+    else
+      :update
+    end
+  else
+    :add
+  end
+end
+
+def add_package_cmd(npm_module_name, module_details)
+  strategy = bundle_strategy(npm_module_name)
+  cmd = ''
+  if @options[:git_packager]
+    cmd += 'RPM_PACKAGER="$(git config user.name) <$(git config user.email)>" '
```

Comment:
I thought `rpmdev-packager` already looked in git by default so I'd expect this to be redundant.

Reply:
Apparently it doesn't. Without that option all my changelogs contained "vagrant".

Reply:
Odd, we should look into that when we include it in obal.